aos_cd_jobs/updater.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+from os import getenv, listdir, rename, walk
+from os.path import join
+from shutil import rmtree
+
+from git import Repo
+
+from aos_cd_jobs.common import JOBS_DIRECTORY, initialize_repo
+
+def update_branches(repo):
+    for job in list_jobs(repo):
+        branch = get_branch_by_name(repo.branches, job)
+        if branch is not None:
```

Comment:
I don't think there is a lot of benefit to this. Without the method, you have

```py
if branch in repo.branches:
    branch.delete()
```

with the method you have

```py
if branch in repo.branches:
    branch = repo.branches[branch]
else:
    branch = None

if branch is not None:
    branch.delete()
```